src/workers/python/python_package.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eux
+
+rm -rf ./package ./package.tar
+mkdir ./package
+pip install -t ./package python-runner friendly_traceback jedi
```

Comment:
You may want to start pinning dependency versions, especially for friendly_traceback